app/api/events.py

```diff
@@ -318,9 +307,8 @@ def query(self, view_kwargs):
                 'is_user_itself', user_id=int(view_kwargs['user_sales_admin_id'])
             ):
                 raise ForbiddenException({'source': ''}, 'Access Forbidden')
-            user = safe_query(
-                db, User, 'id', view_kwargs['user_sales_admin_id'], 'user_sales_admin_id'
-            )
+            user = safe_query(User, 'id', view_kwargs['user_sales_admin_id'],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)